content/docs/examples/multicluster/split-horizon-eds/index.md

```diff
@@ -354,4 +378,5 @@ $ kubectl delete --context=$CTX_CLUSTER1 -f istio-auth.yaml
 $ kubectl delete --context=$CTX_CLUSTER1 ns istio-system
 $ for i in install/kubernetes/helm/istio-init/files/crd*yaml; do kubectl delete --context=$CTX_CLUSTER1 -f $i; done
 $ kubectl delete --context=$CTX_CLUSTER1 ns sample
+$ rm istio-auth.yaml n2-k8s-config
```

Comment:
add CTX_CLUSTER1

Reply:
Fixed in https://github.com/istio/istio.io/pull/3428/commits/25ae2bd2408560d4d13db97c115d3a4313d30d33